multiverse_server:
  host: "tcp://127.0.0.1"
  port: 7000

multiverse_client:
  length_unit: m
  angle_unit: rad
  mass_unit: kg
  time_unit: s
  handedness: rhs
  root_frame_id: map

  services:
    socket:
      port: 7400

  publishers:
    tf:
      port: 7300
      rate: 60
